runner/app/routes/image_to_image.py

```diff
@@ -182,19 +184,21 @@ async def image_to_image(
     has_nsfw_concept = []
     for seed in seeds:
         try:
-            imgs, nsfw_checks = pipeline(
-                prompt=prompt,
-                image=image,
-                strength=strength,
-                loras=loras,
-                guidance_scale=guidance_scale,
-                image_guidance_scale=image_guidance_scale,
-                negative_prompt=negative_prompt,
-                safety_check=safety_check,
-                seed=seed,
-                num_images_per_prompt=1,
-                num_inference_steps=num_inference_steps,
-            )
+            pipeline_call = partial(pipeline, 
+                                    prompt=prompt,
+                                    image=image,
+                                    strength=strength,
+                                    loras=loras,
+                                    guidance_scale=guidance_scale,
+                                    image_guidance_scale=image_guidance_scale,
+                                    negative_prompt=negative_prompt,
+                                    safety_check=safety_check,
+                                    seed=seed,
+                                    num_images_per_prompt=1,
+                                    num_inference_steps=num_inference_steps,
+                                    scheduler=scheduler
```

Comment:
If we go with this solution then scheduler should be defined.

Reply:
This was removed